go.mod

```
@@ -12,14 +12,14 @@ replace (
 	github.com/docker/docker => github.com/docker/docker v20.10.2+incompatible
 	github.com/docker/libnetwork => github.com/docker/libnetwork v0.8.0-dev.2.0.20190624125649-f0e46a78ea34
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.2
+	github.com/googleapis/gax-go/v2 => github.com/googleapis/gax-go/v2 v2.0.5
 	github.com/juju/errors => github.com/k3s-io/nocode v0.0.0-20200630202308-cb097102c09f
 	github.com/k3s-io/helm-controller => github.com/k3s-io/helm-controller v0.11.8
-	github.com/kubernetes-sigs/cri-tools => github.com/k3s-io/cri-tools v1.22.0-k3s1
+	github.com/kubernetes-sigs/cri-tools => github.com/k3s-io/cri-tools v1.21.0-k3s1
```

```suggestion
	github.com/kubernetes-sigs/cri-tools => github.com/k3s-io/cri-tools v1.22.0-k3s1
```
we shouldn't be downgrading this on master/main, i assume this is a mistake

---

I have a script that syncs RKE2 module replacements with the version of k3s that we're importing. That script keeps wanting to downgrade it because k3s release-1.22 is on the wrong version:
https://github.com/k3s-io/k3s/blob/release-1.22/go.mod#L26

But yes it's a mistake.